account_banking_sepa_direct_debit/tests/test_sdd.py

```diff
@@ -7,18 +7,36 @@
 from lxml import etree
 
 from odoo import fields
+from odoo.tests.common import SavepointCase
 from odoo.tools import float_compare
 
-from odoo.addons.account.tests.account_test_multi_company_no_chart import (
-    TestAccountMultiCompanyNoChartCommon,
-)
 
-
-class TestSDDBase(TestAccountMultiCompanyNoChartCommon):
+class TestSDDBase(SavepointCase):
     @classmethod
     def setUpClass(cls):
         super().setUpClass()
         self = cls
+        self.company_B = self.env["res.company"].create({"name": "Company B"})
```

Comment:
The `self = cls` was a trick for not creating a big diff when the switch to SavepointCase was onde, but you shouldn't use `self` in new lines, and don't change it in existing ones. If any thing to do, is the contrary one:  change all existing `self` to `cls` and remove the alias.